src/v/cloud_storage/anomalies_detector.h

```diff
@@ -48,15 +48,35 @@ class anomalies_detector {
         scrub_status status{scrub_status::full};
         std::optional<model::offset> last_scrubbed_offset;
         anomalies detected;
-        int32_t ops{0};
+        int32_t get_ops{0};
+        int32_t segments_visited{0};
 
         result& operator+=(result&&);
     };
 
+    using segment_depth_t = named_type<int32_t, struct segment_depth_t>;
```

Comment:
is it OK to have both type name and tag type name match?

Reply:
I think so but here it's def a typo